modules/weboramaRtdProvider.md

```diff
@@ -0,0 +1,72 @@
+# Weborama Real-Time Data Submodule
+
+```
+Module Name: Weborama Rtd Provider
+Module Type: Rtd Provider
+Maintainer: prebid-support@weborama.com
+```
+
+# Description
+
+Weborama provides a Semantic AI Contextual API that classify in Real-time a web page seen by a web user within generic and custom topics. It enables publishers to better monetize their inventory and unlock it to programmatic.
+
+ORTB2 compliant and FPD support for Prebid versions < 4.29
+
+Contact prebid-support@weborama.com for information.
+
+### Publisher Usage
+
+Compile the Weborama RTD module into your Prebid build:
+
+`gulp build --modules=rtdModule,weboramaRtdProvider`
+
+Add the Weborama RTD provider to your Prebid config.
+
+```javascript
+pbjs.setConfig(
+    ...
+    realTimeData: {
+        auctionDelay: 1000,
+        dataProviders: [
+            {
+                name: "weborama",
+                waitForIt: true,
+                params: {
+                  weboCtxConf: {
+                      setTargeting: true,
+                      token: "<<token provided by weborama>>",
+                      targetURL: "..." // default is document.URL
+                    }
+                }
+            }
+        ]
+    }
+    ...
+}
+```
+
+### Parameter Descriptions for the Weborama Configuration Section
+
+| Name  |Type | Description   | Notes  |
+| :------------ | :------------ | :------------ |:------------ |
+| name | String | Real time data module name | Mandatory. Always 'Weborama' |
+| waitForIt | Boolean | Mandatory. Required to ensure that the auction is delayed until prefetch is complete | Optional. Defaults to false but recommended to true |
+| params | Object | | Optional |
+| params.weboCtxConf | Object | Weborama Contextual Configuration | Optional |
+| params.weboCtxConf.token | String | Security Token provided by Weborama, unique per client | Mandatory |
+| params.weboCtxConf.targetURL | String | Url to be profiled in the contextual api | Optional. Defaults to `document.URL` |
+| params.weboCtxConf.defaultProfile | Object | default value of the profile to be used when there are no response from contextual api (such as timeout)| Optional. Default is `{}` |
+| params.weboCtxConf.gamTargetingWeboCtxKey | String | allow rename the key `webo_ctx`  in gam targeting| Optional. Default is `webo_ctx`|
+| params.weboCtxConf.gamTargetingWeboDSKey | String | allow rename the key `webo_ds` in gam targeting | Optional. Default is `webo_ds`|
+| params.weboCtxConf.setTargeting|Boolean|If true, will use the contextual profile to set the gam targeting of all adunits managed by prebid.js| Optional. Default is *true*.|
+| params.weboCtxConf.setOrtb2|Boolean|If true, will use the contextual profile to set the ortb2 configuration on `site.ext.data`| Optional. Default is *true*.|
```

Comment:
why not register for a segtax and set site.data ?

Reply:
The API is still in development, we will probably do it in a near future

Reply:
we decide to keep this feature not active by default. clientes will use at their own responsibility